configs/stash-cache/config.d/50-stash-cache-paths.cfg

```diff
@@ -8,9 +8,6 @@
 # This file is part of the StashCache Daemon
 # https://opensciencegrid.org/docs/data/stashcache/overview/
 
-# The directory on local disk where files will be cached, e.g. "/mnt/stash".
-oss.localroot $(rootdir)
```

Comment:
Where will this get specified if you remove this line?

Reply:
https://github.com/opensciencegrid/Software-Redhat/blob/trunk/osg-xrootd/osg/50-osg-paths.cfg#L17-L18